src/video-player.js

```diff
@@ -495,7 +495,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
       const error = this.videojs.error();
       if (error) {
         let type = this.videojs.cloudinary.currentSourceType();
-        if ((error.code === 4 || error.code === 25) && (type === 'VideoSource' || type === 'AudioSource')) {
+        if (error.code === 4 && (type === 'VideoSource' || type === 'AudioSource')) {
```

Comment:
is error code 25 handled elsewhere? what does it mean? can you please add a comment explaining why code 4 is handled here this way?

Reply:
error 25 was a quick fix / workaround for a bug